In [ ]:
import pandas as pd

path = "C:\PERKULIHAAN\SEMESTER 5\CERTAN\CERTAN\week 2\GlobalWeatherRepository.csv"
df = pd.read_csv(path)

df[['temperature_fahrenheit',
   'pressure_mb',
   'air_quality_Sulphur_dioxide']].corr()

,temperature_fahrenheit,pressure_mb,air_quality_Sulphur_dioxide
temperature_fahrenheit,1.000000,-0.486833,-0.023152
pressure_mb,-0.486833,1.000000,0.018620
air_quality_Sulphur_dioxide,-0.023152,0.018620,1.000000


# Tugas 1
## Analisis Korelasi Dataset Global Weather

### Tujuan
Menganalisis hubungan atau korelasi antara beberapa variabel pada dataset Global Weather.

### Variabel yang Dianalisis
- `temperature_fahrenheit`
- `pressure_mb`
- `air_quality_Sulphur_dioxide`

Berdasarkan hasil analisis korelasi, hubungan yang paling terlihat terdapat antara temperature_fahrenheit dan pressure_mb, dengan nilai korelasi -0.486833. Sementara itu, hubungan antara temperatur dengan Sulphur Dioxide dan tekanan dengan Sulphur Dioxide sangat lemah.